map/drawMap.php

```diff
@@ -341,17 +341,16 @@ protected function loadOrderArrows()
 	{
 		foreach($this->orderArrows as &$orderArrow)
 		{
-			foreach($orderArrow as $name => &$param)
-			{
-				if ( $name == 'headAngle' ) $param = M_PI/$param;
+			foreach($orderArrow as $name => &$param) {

```

Comment:
Can each of the opening and closing brackets within this loop live on their own line to conform with rest of the style throughout the codebase?